wagtail/actions/copy_for_translation.py

```diff
@@ -146,7 +148,11 @@ def execute(self, skip_permission_checks=False):
         self.check(skip_permission_checks=skip_permission_checks)
 
         translated_page = self._copy_for_translation(
-            self.page, self.locale, self.copy_parents, self.alias, self.exclude_fields
+            self.page.live and self.page or self.page.get_latest_revision_as_page(),
```

Comment:
`self.page if self.page.live else self.page.get_latest_revision_as_page()` is the more familiar syntax I think, but happy with this logic.